crypto/CMakeLists.txt

```diff
@@ -148,6 +148,13 @@ if(${ARCH} STREQUAL "aarch64")
 
     chacha/chacha-armv8.${ASM_EXT}
     test/trampoline-armv8.${ASM_EXT}
+    ${PROJECT_SOURCE_DIR}/third_party/s2n-bignum/Arm/bignum_add_p384.S
```

Comment:
Maybe we can include all the assembly files with:
`${PROJECT_SOURCE_DIR}/third_party/s2n-bignum/Arm/*.S`